trivy_report/report_generator.py

```diff
@@ -81,7 +101,13 @@ def parse_results(data: ReportDict, existing_issues: List[str]) -> Iterator[Repo
     :param data: The report data that was parsed from JSON file.
     :param existing_issues: List of GitHub issues, used to exclude already reported issues.
     """
-    results = data["Results"]
+
+    try:
+        results = data["Results"]
+    except Exception as e:
+        raise KeyError(
```

Comment:
It is probably better to return just False or None here (Function signature must be changed) and then check this in the function caller.